pkg/nsx/services/common/types.go

```diff
@@ -122,3 +122,14 @@ type VPCResourceInfo struct {
 	ID       string
 	ParentID string
 }
+
+type VPCNetworkConfigInfo struct {
```

Comment:
Dose this VPCNetworkConfigInfo struct will be used other modules except VPC, if not, we shoud not put it in the common service file.